wa/framework/execution.py

```diff
@@ -491,8 +491,9 @@ def finalize_run(self):
         self.logger.info('Finalizing run')
         self.context.end_run()
         self.pm.enable_all()
-        self.pm.process_run_output(self.context)
-        self.pm.export_run_output(self.context)
+        with signal.wrap('OVERALL_RESULTS_PROCESSING'):
```

Comment:
Should  rename this to `'RUN_OUTPUT_PROCESSED'`

Reply:
Updated.